code/tracing/FrameProfiler.cpp

```diff
@@ -136,6 +136,17 @@ void FrameProfiler::processEvent(const trace_event* event) {
 		return;
 	}
 
+	if (_mainThreadID == -1) {
+		// We need the ID of the main thread to filter out multi threaded events. We just assume that the first event
+		// of the main process we see it from the main thread. That should be a safe assumption.
```

Comment:
Minor typo: "we see it from the main thread" looks like it should be "we see **is** from the main thread".